pkg/constants/constants.go

```diff
@@ -40,7 +40,9 @@ var (
 
 // KFService Annotations
 var (
-	KFServiceGKEAcceleratorAnnotationKey = KFServingAPIGroupName + "/gke-accelerator"
+	KFServiceGKEAcceleratorAnnotationKey             = KFServingAPIGroupName + "/gke-accelerator"
+	KFServiceModelProvisioningSourceURIAnnotationKey = KFServingAPIGroupName + "/provisioning/sourceURI"
+	KFServiceModelProvisioningMountPathAnnotationKey = KFServingAPIGroupName + "/provisioning/mountPath"
```

Comment:
This is an odd pattern for annotations. We should probably chat offline about the annotation and defaulting behavior.

Reply:
Have made it private as discussed